src/ring_theory/euclidean_domain.lean

```diff
@@ -29,14 +29,12 @@ open euclidean_domain set ideal
 
 section gcd_monoid
 
-variables {R : Type*} [euclidean_domain R] [gcd_monoid R]
+variables {R : Type*} [euclidean_domain R] [gcd_monoid R] {p q : R}
 
-lemma gcd_ne_zero_of_left (p q : R) (hp : p ≠ 0) :
-  gcd_monoid.gcd p q ≠ 0 :=
+lemma gcd_ne_zero_of_left (hp : p ≠ 0) : gcd_monoid.gcd p q ≠ 0 :=
```

Comment:
Why did you make `q` implicit here?

Reply:
This matches the two lemmas right under.

Reply:
I'd argue the change should be in the other direction.

Reply:
Well, I can keep `q` explicit if you insist. But this lemma is only used inside this file, and never with explicit arguments.

Reply:
Let's either keep `q` explicit or not touch these lemmas at all.

Reply:
I think it's fine to have these implicit.

bors merge